src/@commonsku/styles/Popup.tsx

```diff
@@ -193,4 +206,17 @@ export const ShowPopup: React.FC<Omit<PopupProps, 'onClose'> & {
   </>
 }
 
+const ChevronPopupWindow = (props) => {
+  const { onPreviousButton, onNextButton, index, max } = props;
+  return <>
+    {index > 0 && <ChevronButton direction='left' left={true} onClick={() => onPreviousButton(index)} />}
+    <PopupWindow {...props} />
+    {index < max && <ChevronButton direction='right' onClick={() => onNextButton(index)} />}
```

Comment:
better to pass event function directly: `onClick={onNext}`,  so it can have event argument.  
And we may not care about index and max.